building/devices/zynqmp.rst

```diff
@@ -0,0 +1,115 @@
+.. _zynqmp_zcu102:
+
+#######################
+QEMU for ZynqMP zcu102
+#######################
+The instructions below show how to run OP-TEE using QEMU for ZynqMP zcu102
```

Comment:
s/show/shows/

Reply:
'instructions ... show' therefore no 's', isn't it? i'm going to remove 'the' before 'instructions'